### PR TITLE
Created CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by AdrianCollado**
This file lists the contributing guidelines for Solum.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- Create an issue if one does not exist.
+  - Clearly describe the issue, including steps to reproduce if the issue is a bug report.
+- Fork the repository on GitHub to your own account.
+
+## Making Changes
+- Create a new branch with the necessary changes.
+  - Name it according to what you need to accomplish.
+    - Features can be prefixed with `feature/`
+    - Bugs can be prefixed with `bug/`
+- Make commits into logical units.
```
Comment (JohnBehnke) [inline CONTRIBUTING.md]: This could be explained better

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# How To Contribute
+## Getting Started
+- Make sure you have a GitHub account
+- Create an issue if one does not exist.
+  - Clearly describe the issue, including steps to reproduce if the issue is a bug report.
+- Fork the repository on GitHub to your own account.
+
+## Making Changes
```
Comment (kelleyb) [inline CONTRIBUTING.md]: This should contain a link to the style guide

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- Create a new branch with the necessary changes.
+  - Name it according to what you need to accomplish.
```
Comment (kelleyb) [inline CONTRIBUTING.md]: Poorly worded. Maybe it should be "Name it _based on_ what you need to accomplish"?

Comment (JohnBehnke) [inline CONTRIBUTING.md]: It could also use a good meme 👌

File: CONTRIBUTING.md (hunk 2 above)
Comment (AdrianCollado) [inline CONTRIBUTING.md]: Good Idea!

File: CONTRIBUTING.md (hunk 3 above)
Comment (AdrianCollado) [inline CONTRIBUTING.md]: That's a very good point.

**Comment by kelleyb** [comment]
LGTM! Ship it!

```
              |    |    |                 
             )_)  )_)  )_)              
            )___))___))___)\            
           )____)____)_____)\\
         _____|____|____|____\\\__
---------\                   /---------
  ^^^^^ ^^^^^^^^^^^^^^^^^^^^^
    ^^^^      ^^^^     ^^^    ^^
         ^^^^      ^^^
```

**Pull request merged**